### ISSUE TITLE
siren-tests: fix the screenshot 

### ISSUE THREAD

**Issue by rayakurl** [scylladb/argus#270]
Currently the screenshot in case of a test failure in siren-tests job is not displayed.

Cloud qa team has a new aqs account  - Account: sc-dbaas-data-qa (400694452550)
 we can probably use for Argus, just specify which permissions are needed and I can take it with @d-helios.

**Comment by rayakurl** [comment]
@roydahan , @temichus - FYI

**Comment by roydahan** [comment]
1. I don't understand the issue exactly, how is the screenshot related to the permissions of AWS account.
2. It sounds like the estimation by @k0machi of complexity and time are quite high.

I suggest we think of other directions, e.g. setting up a grafana proxy and use the same way we use today.

**Comment by fruch** [comment]
Maybe we should revisit the decision to make those screenshots private ? like anyone with a user on scylla-cloud could see those pages, nothing secret on those pages (maybe the broken one might be a bit embarrassing)

**Comment by rayakurl** [comment]
@fruch - I think that since the cloud project is private, we should keep it the same. The screenshot is taken only in case of a failure, so high chance that it won't look good.

**Comment by fruch** [comment]
> @fruch - I think that since the cloud project is private, we should keep it the same. The screenshot is taken only in case of a failure, so high chance that it won't look good.

I don't think we care about things that don't look good, the question is it a security concern or not.

Not the test code, but the screenshot of a system that is open to the world anyhow.

**Comment by roydahan** [comment]
We can start by making them public.
(Can we make it only for organization Scylla?)

**Comment by roydahan** [comment]
@k0machi is it possible for you to just include the S3 link to the screenshot instead of displaying the screenshot?
If so, we can provide that and whoever want to see it will click on the link, provide his AWS credentials and download it.

Otherwise, @rayakurl you'll need to change the bucket to be public.

**Comment by k0machi** [comment]
@roydahan It should work in theory to just point to the bucket/screenshot page in aws console, however link/image doesn't matter, the screenshot needs tokens generated by AWS to be accessed via direct url.

**Comment by roydahan** [comment]
Yes, but all people from our organization should be able to reach that bucket through the link with their AWS credentials.

**Comment by rayakurl** [comment]
@k0machi , @fruch - do we have any plans to fix this issue?  we are still using Argus

**Comment by fruch** [comment]
> @k0machi , @fruch - do we have any plans to fix this issue?  we are still using Argus

As @roydahan mentioned before 
It's or the bucket would be set as public SCT, and then it would work.

Or the best we'll be able to try is to supply a clickable link that if a user is logged on it could bring him into the AWS console seeing that object.

We are not going to invest in finding a way to expose pictures from private buckets

**Comment by rayakurl** [comment]
@fruch so will you supply the clickable link?

**Comment by fruch** [comment]
> @fruch so will you supply the clickable link?

the quickest way would be to set the bucket public

@k0machi please estimate the work needed
